plugin/geoip/setup_test.go

```diff
@@ -0,0 +1,114 @@
+package geoip
+
+import (
+	"fmt"
+	"net"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/coredns/caddy"
+	"github.com/coredns/coredns/core/dnsserver"
+)
+
+var (
+	_, file, _, _    = runtime.Caller(0)
+	fixturesDir      = filepath.Join(filepath.Dir(file), "testdata")
```

Comment:
you can just open files in `testdata` relative from the tests, no need for `runtime.Caller(0)` stuff

Reply:
Done